pkg/sqs/sqs_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build test
```

Comment:
We only exclude e2e tests from the build. This is a unit test, please remove this line. For some reason I was not able to run this unit test from my local :(

Reply:
> We only exclude e2e tests from the build. This is a unit test, please remove this line. For some reason I was not able to run this unit test from my local :(

May be some problem with docker in your machine